src/ingress/management/commands/disable_parser.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+
+from ingress.tools import Endpoint
+
+
+class Command(BaseCommand):
+    help = "Disables the parser for an endpoint (if the parsers exist)."
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'url_key',
+            help="The url_key for the endpoint. Use the 'list_endpoints' command to show the url_keys for "
+                 "the existing endpoints.")
+
+    def handle(self, *args, **options):
+        url_key = options['url_key']
+
+        endpoint = Endpoint.objects.get(url_key=url_key)
+        if endpoint.parser_enabled == False:
```

Comment:
`if not endpoint.parser_enabled` is meer _pytonic_.

Reply:
tenzij je heel expliciet False wilt checken, en waardes zoals None of 0 iets anders moeten doen. Maar omdat dit een database boolean is, zal het altijd True/False zijn.

Reply:
Klopt. Had ik er eerst ook staan, maar ik wilde hier echt expliciet zijn. Maar je hebt gelijk, veel voegt het niet toe.